index.html

```diff
@@ -37,9 +43,235 @@ <h1>Donice Ubaru</h1>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section>
+
+    <h2>About</h2>
+    <div id="About-myself">
+      <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like
+        what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <button type="button">
+        Get my CV
+        <span class="material-symbols-outlined">download</span>
+      </button>
+
+      <div class="skill">
+        <div class="skill-card" id="language">
+          <h4>Languages</h4>
+          <ul>
+            <li>Javascript(ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+
+        <div class="skill-card" id="frameworks">
+          <h4>Frameworks</h4>
+          <ul>
+            <li>React.js</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+          </ul>
+        </div>
+
+        <div class="skill-card" id="skills">
+          <h4>Skills</h4>
+          <ul>
+            <li>Database Management</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Version Control</li>
+            <li>Web Development</li>
+          </ul>
+        </div>
+
+      </div>
+    </div>
+  </section>
+
+
+
+  <section><img src="./images/SideImage.svg" id="nav-image" alt="" />
+    <div id="contact-me">
+
+      <h1>Get Started</h1>
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+
+      <form action="/">
+        <div class="form">
+          <label for="name">Your name</label><br>
+          <input id="name" type="text" placeholder="Donice Ubaru">
+          <label for="email">Your email</label><br>
+          <input id="email" type="text" placeholder="doniceubaru1@gmail.com">
+          <label for="message">Your mesaage</label><br>
+          <textarea id="message" rows="4" cols="50" placeholder="Enter yout new message here"></textarea>
+        </div>
+
+        <button type="button">Start Collaboration</button>
+      </form>
+    </div>
   </section>
 
+  <footer>
+    <a href="/">Get my Resume<span class="material-symbols-outlined">
+        open_in_new
+      </span></a>
+    <div class="socials">
+      <a href="">
+        <img src="./images/icons/ic_angellist.svg" alt="angel list">
+      </a>
+      <a href="">
+        <img src="./images/icons/ic_github.svg" alt="angel list">
+      </a>
+      <a href="">
+        <img src="./images/icons/ic_linkedin.svg" alt="angel list">
+      </a>
+      <a href="">
+        <img src="./images/icons/ic_medium.svg" alt="angel list">
+      </a>
+      <a href="">
+        <img src="./images/icons/ic_twitter.svg" alt="angel list">
+      </a>
+      <a href="">
+        <img src="./images/icons/ic_stackoverflow.svg" alt="angel list">
+      </a>
+    </div>
```

Comment:
- [ ] Kindly ensure that the list of icons is an `a` tag wrapped in a `li` tag then all wrapped in a `ul` tag.

something like this 

```
<ul>
  <li><a>home</a></li>
  <li><a>house</a></li>
  <li><a>yard</a></li>
  <li><a>apartment</a></li>
</ul>
```

This is good for semantic and user experience reasons respectively and also enables one to connect with the icons using the `href` attribute.